www/test/functional/report.spec.js

```diff
@@ -1,3 +1,4 @@
+var moment = require('moment');
```

Comment:
I added `moment` to deal better with dates. In a next moment, I'll put this library in another parts of the project.